app/lib/facet_query_builder.rb

```diff
@@ -0,0 +1,32 @@
+class FacetQueryBuilder
+  def initialize(facets:)
+    @facets = facets
+  end
+
+  def call
+    dynamic_facets.reduce({}) { |query, facet|
+      # TODO title will only work for Orgs, this key will need changed for
+      # other dynamic facets
+      #
+      # "1000,order:value.title" is specifying that we want 1000 results back
+      # which are ordered by the title attribute of each value (option)
+      # that is returned
+      query.merge(facet.key => "1000,order:value.title")
```

Comment:
(not a blocker, pedantry ahoy)

This is a good comment :+1:, but a lot of what it explains could be expressed in code too, something like:
```
num_facets = 1000
order_field = 'value.title'
query.merge(facet.key => "#{num_facets},order:#{order_field}")
```

(if we had more facet key/val pairs it might be worth making that a hash, but we just have the one for now, so it didn't seem worth overcomplicating it)

Personally I find it more readable, and it reduces the chance a comment would go out of sync with the code it's describing.